### ISSUE TITLE
JAWS bug: Elements in dialog associated with aria-describedby read even if hidden

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/a11y-research#33]
Using JAWS, elements which are created dynamically in a dialog are read even if they are marked as hidden. This is a JAWS bug and VFO should be notified.

Example JSFiddle with the bug:
https://jsfiddle.net/8q933y29/6/

**Comment by jessegreenberg** [comment]
Found in https://github.com/phetsims/john-travoltage/issues/226

**Comment by jessegreenberg** [comment]
For reference, JAWS reads this element in https://jsfiddle.net/8q933y29/8/:

```html
<p aria-hidden="true" style="display: none;" hidden>This is a hidden description.  Will AT find me?</p>
```

**Comment by jessegreenberg** [comment]
This is actually the way ```aria-describedby``` is supposed to work, see https://www.paciellogroup.com/blog/2015/05/short-note-on-aria-labelledby-and-aria-describedby/.

This is not a bug, closing.
